docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -116,6 +116,8 @@ Experimental features might be changed or removed without prior notice.
 | `disableSecretsCompatibility`               | Disable duplicated secret storage in legacy tables                                                                                                                                                                                                                                |
 | `logRequestsInstrumentedAsUnknown`          | Logs the path for requests that are instrumented as unknown                                                                                                                                                                                                                       |
 | `unifiedStorage`                            | SQL-based k8s storage                                                                                                                                                                                                                                                             |
+| `dualWritePlaylistsMode2`                   | Enables dual writing of playlists to both legacy and k8s storage in mode 2                                                                                                                                                                                                        |
```

Comment:
If we have one feature toggle per entity + mode, the list will be even bigger... what do you think of instead of having a boolean, we have something like `PlaylistMode=<a number>` ?

Reply:
For now only booleans but eventually string/integer. Unfortunately, I think we need feature flags one way or the other because we can customise which feature flags we set based on deployment wave but not config.